src/main/resources/reference.conf

```diff
@@ -1207,6 +1222,32 @@ resourceTypes = {
     }
     reuseIds = true
   }
+  landing-zone = {
+    actionPatterns = {
+      delete = {
+        description = "delete this landing zone"
+      }
+      list_resources = {
+        description = "list resources in this landing zone"
+      }
+      read_policies = {
+        description = "view all policies and policy details for this landing zone"
+      }
+      set_parent = {
```

Comment:
I got a little tripped up that I apparently need the magic `set_parent` and `add_child` actions in order for the hierarchical resource to work.

Reply:
I am setting the parent as part of the resource creation call, I think I still got an except without `set_parent` though. Not 100% sure, will double check..

Reply:
ok, [I think you are correct](https://github.com/broadinstitute/sam/blob/develop/src/main/scala/org/broadinstitute/dsde/workbench/sam/api/SecurityDirectives.scala#L77), `set_parent` is required. But I still think you should set the parent on creation. If you allow `remove_child` from the parent then the LZ can be orphaned or moved which I don't think we want.

Reply:
Ok cool, I will keep `set_parent`, remove `remove_parent`, and set the parent at resource creation. 👍